.github/workflows/backend-test.yml

```diff
@@ -22,12 +22,11 @@ jobs:
       - name: Install codecov
         working-directory: backend
         run: pip install codecov
-
       - name: pytest
         working-directory: backend
         run: |
             set +e
-            OUTPUT=$(poetry run py.test --cov-report xml --cov=. --cov-fail-under 100)
+            OUTPUT=$(poetry run pytest */tests/ --cov-report xml --cov=. --cov-fail-under 100)
```

Comment:
I think this could be `.`, as pyrest only gets files starting with `test_` :)

Reply:
This way  integration tests are not run